### PR TITLE
Passing additional params into query string

### PR DESCRIPTION
**Pull request by XiaoxuHU**

### CODE REVIEW

**Comment by XiaoxuHU** [comment]
I did some formatting, please ignore. 
Change I did is for **selectDetector** and **selectDetectorNewTab** method for merging additional parameters from renderingProperties with query parameters and pass merged parameters for navigation/open new tab

File: AngularApp/.npmrc [hunk 1]
```diff
@@ -1,3 +0,0 @@
-registry=https://pkgs.dev.azure.com/app-service-diagnostics-portal/_packaging/app-services-diagnostics-portal-nuget-feed/npm/registry/
```
Comment (XiaoxuHU) [inline AngularApp/.npmrc:1]: Redo for [removing npmrc file](https://github.com/Azure/Azure-AppServices-Diagnostics-Portal/pull/1547), somehow it get back by [Nazeef's PR](https://github.com/Azure/Azure-AppServices-Diagnostics-Portal/pull/1520), checked with him this is git issue

File: AngularApp/projects/diagnostic-data/src/lib/components/detector-list/detector-list.component.ts [hunk 2]
```diff
@@ -407,32 +405,33 @@ export class DetectorListComponent extends DataRenderBaseComponent {
         // Log children detectors click
         this.logEvent(TelemetryEventNames.ChildDetectorClicked, clickDetectorEventProperties);
         this.queryParams = UriUtilities.removeChildDetectorStartAndEndTime(this._activatedRoute.snapshot.queryParams);
+        const additionalParams = this.renderingProperties.additionalParams ? JSON.parse(this.renderingProperties.additionalParams) : {};
+        const combinedParams = { ...this.queryParams, ...additionalParams };
```
Comment (cindylovescoding) [inline AngularApp/projects/diagnostic-data/src/lib/components/detector-list/detector-list.component.ts:409]: What will happen if there are duplicate items in queryParams and additionalParams?

Comment (XiaoxuHU) [reply AngularApp/projects/diagnostic-data/src/lib/components/detector-list/detector-list.component.ts:409]: For same key, value from addtionalParams will override the one from queryParams.

**Review by cindylovescoding — APPROVED** [review]

**Pull request merged**